src/controllers/PurseController.php

```diff
@@ -135,7 +137,18 @@ public function generateDocument($action, $params)
         try {
             $data = Purse::perform($action, $params);
         } catch (ResponseErrorException $e) {
-            Yii::$app->getSession()->setFlash('error', Yii::t('hipanel:finance', 'Failed to generate document! Check requisites!'));
+            $contactUrl = Html::a(Url::toRoute(['@requisite/view', 'id' => $e->getResponse()->getData()['_error_ops']['requisite_id']], true),
+                ['@requisite/view', 'id' => $e->getResponse()->getData()['_error_ops']['requisite_id']]);
+            $type = $e->getResponse()->getData()['_error_ops']['type'];
+
+            if (Yii::$app->user->can('requisites.update')) {
+                Yii::$app->getSession()->setFlash('error',
+                    Yii::t('hipanel:finance', "No templates for requisite. Follow this link") . ' ' . $contactUrl . ' ' .
+                    Yii::t('hipanel:finance', "and set template of type") . " '$type'");
```

Comment:
```suggestion
                Yii::$app->getSession()->setFlash('error',
                    Yii::t('hipanel:finance', "No templates for requisite. Follow this link") . ' ' . $contactUrl . ' ' .
                    Yii::t('hipanel:finance', "and set template of type") . " '$type'");
```

Do not split message into pieces, use message parameters instead: https://www.yiiframework.com/doc/guide/2.0/en/tutorial-i18n#message-parameters